fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -217,10 +217,10 @@
 
     private static final Logger LOGGER = getLogger(FedoraLdpIT.class);
 
-    private static DateTimeFormatter headerFormat =
+    private static final DateTimeFormatter headerFormat =
       DateTimeFormatter.ofPattern("EEE, dd MMM yyyy HH:mm:ss zzz", Locale.US).withZone(of("GMT"));
```

Comment:
This looks to be an RFC-1123 date format. So I think you could use the predefined one in the DateTimeFormatter or you could use the VersionService's which has the timezone set too.

[org.fcrepo.kernel.api.services.VersionService.MEMENTO_RFC_1123_FORMATTER](https://github.com/fcrepo4/fcrepo4/blob/master/fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/services/VersionService.java#L56)